packages/audit/tests/test_event_logger.py

```
@@ -0,0 +1,172 @@
+        )
+
+        assert record["tid"] == "my-trace"
+        assert record["ptid"] == "parent-trace"
+        assert record["aid"] == "agent-123"
+        assert record["ctx"]["skill"] == "issue-fix"
+
+    def test_multiple_events_append(self, tmp_path: object) -> None:
+        project_dir = str(tmp_path)
+        os.makedirs(os.path.join(project_dir, ".claude"), exist_ok=True)
+
+        for i in range(3):
```

Ruff (0.15.9)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)